.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
+
+defaults: &defaults
+  working_directory: ~/repo
+  docker:
+    - image: circleci/node:8.11
+
+jobs:
+  publish-version:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          - v1-dependencies-
+
+      - attach_workspace:
+          at: ~/repo
+      - run:
+          name: Authenticate with registry
+          command: echo "//registry.npmjs.org/:_authToken=$npm_TOKEN" > ~/repo/.npmrc
+
+      - run: yarn version --new-version $CIRCLE_BRANCH
```

Comment:
if we follow git branching name, it won't create proper version here

Reply:
That's why I filter branch name with following regex: `/^v.*/`. I think we should talk about what flow do we prefer